packages/react-image/src/components/Image/Image.types.ts

```diff
@@ -2,6 +2,11 @@ import * as React from 'react';
 import { ComponentPropsCompat } from '@fluentui/react-utilities';
 
 export interface ImageProps extends ComponentPropsCompat, React.ImgHTMLAttributes<HTMLImageElement> {
+  /**
+   * An image can have source URL.
+   */
+  src?: string;
```

Comment:
moved up as that's the most important prop of the component

Reply:
This prop already exists in `React.ImgHTMLAttributes<HTMLImageElement>`, Is the declaration here only for explicity control generation ?

Reply:
yeah I mentioned that in this RFC https://github.com/microsoft/fluentui/pull/18935#discussion_r671116750.

- also react-dom doesn't provide any description 🍤

Reply:
I don't think we need descriptions for native attributes. I think it should be safe to remove since you already have explicit controls declared in the stories ?

Not blocking though, but we don't generally document native attributes (AFAIK) unless they have a specific meaning for the component that is not covered by a native HTML element. WDYT ?

Reply:
sounds good to me !

> since you already have explicit controls declared i

that's required no matter if we define them or not, so we can provide proper playground for modifying `src` to render any image etc